run.py

```diff
@@ -85,6 +85,10 @@ def _killGroup(signal=SIGTERM):
             print "Error: The specified configuration file doesn't exist."
             exit(1)
 
+        if not exists('/proc'):
+            print "Error: procfs not available on this system."
+            exit(1)
```

Comment:
Much better, exit with a different value than 1 though, that way we can tell why it failed from the outside if we want to.